www/src/App.tsx

```diff
@@ -1,21 +1,54 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+// Libraries
+import React, { FC } from 'react';
+import { Route, BrowserRouter as Router, Switch } from 'react-router-dom';
 
-function App(): JSX.Element {
+// Material UI
+import { Container, CssBaseline, ThemeProvider } from '@material-ui/core';
+
+// Components and Routes
+import { Header, Section } from './components/Header';
+import { Landing } from './routes/Landing';
+import { ResumeReview } from './routes/ResumeReview';
+import { MockInterview } from './routes/MockInterview';
+import { Community } from './routes/Community';
+
+// Styles
+import theme from './styles/theme';
+
+// Constants
+import { COMMUNITY, COMPE_PLUS, MOCK_INTERVIEW, RESUME_REVIEW } from './util/constants';
+
+const header_sections: Section[] = [
+    { title: RESUME_REVIEW, url: '/resume-review' },
+    { title: MOCK_INTERVIEW, url: '/mock-interview' },
+    { title: COMMUNITY, url: '/community' },
+];
+
+const App: FC = () => {
     return (
-        <div className="App">
-            <header className="App-header">
-                <img src={logo} className="App-logo" alt="logo" />
-                <p>
-                    Edit <code>src/App.tsx</code> and save to reload.
-                </p>
-                <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-                    Learn React
-                </a>
-            </header>
-        </div>
+        <ThemeProvider theme={theme}>
+            <CssBaseline />
+            <Container maxWidth={false} style={{ padding: 0 }}>
+                <Header sections={header_sections} title={COMPE_PLUS} />
+                <Router>
+                    <Switch>
+                        <Route path='/resume-review'>
+                            <ResumeReview />
+                        </Route>
+                        <Route path='/mock-interview'>
+                            <MockInterview />
+                        </Route>
+                        <Route path='/community'>
+                            <Community />
+                        </Route>
+                        <Route path='/'>
+                            <Landing />
+                        </Route>
```

Comment:
Nit: Can this be first in the switch?

Reply:
react-router will match with the first thing it can find. So when we go to the `/resume-review` route with that `/` Route at the very beginning of the `Switch`, it will match with that route (I think with the '/') and will be taken to the Landing page. 
I quickly tested this and got that behaviour.

Reply:
We can either do this or if you really want it first we can add `<Route exact path='/'>`. An advantage of not using `exact` is any non existing page will just go to `Landing`

Reply:
I'm gonna opt to keep it as is for now just to have that catchall of leading back to `Landing`

Reply:
Yep I agree with you. Was not aware of that behaviour.